### ISSUE TITLE
Java's Wicket - Verification Failed

### ISSUE THREAD

**Issue by LadyMozzarella** [TechEmpower/FrameworkBenchmarks#1400]
As of Round 10, Preview 3 Wicket is failing, but passes locally.

* [wicket](https://github.com/TechEmpower/TFB-Round-10/tree/master/peak/linux/results-2015-03-06-peak-preview3/latest/logs/wicket) (single query, multiple queries, fortunes, data updates):

  ```bash
[INFO] Building war: /home/techempower/FrameworkBenchmarks/frameworks/Java/wicket/target/hellowicket-1.0-SNAPSHOT.war
[INFO] WEB-INF/web.xml already added, skipping
```

**Comment by martin-g** [comment]
X-Ref: #1403

**Comment by LadyMozzarella** [comment]
Closed in favor of issue #1403.
